docker/build_artifacts/sagemaker/tensorflow-serving.js

```diff
@@ -77,10 +77,12 @@ function tfs_json_request(r, json) {
             body = body.replace("\\'instances\\'", "'instances'")
         }
 
-        var content_types = accept.trim().replace(" ", "").split(",")
-        if (content_types.includes('application/jsonlines') || content_types.includes('application/jsons' == accept)) {
-            body = body.replace(/\n/g, '')
-            r.headersOut['Content-Type'] = content_types[0]
+        if (accept != undefined) {
```

Comment:
Did you perhaps mean to check if accept is not None? - I don't see undefined being neither a keyword nor a variable defined somewhere in the scope.